stdlib/symtable.pyi

```diff
@@ -52,9 +49,8 @@ class Function(SymbolTable):
     def get_nonlocals(self) -> tuple[str, ...]: ...
 
 class Class(SymbolTable):
-    if sys.version_info < (3, 16):
```

Comment:
Out of scope for this PR, but I removed this check, as it's a long time to 3.16 and who knows what will happen until then.